appveyor.yml

```diff
@@ -26,8 +29,7 @@ before_build:
         } Else {
             $env:SHARED="no-shared no-makedepend"
         }
-    - ps: $env:VSCOMNTOOLS=(Get-Content ("env:VS" + "$env:VSVER" + "0COMNTOOLS"))
-    - call "%VSCOMNTOOLS%\..\..\VC\vcvarsall.bat" %VCVARS_PLATFORM%
+    - call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\vcvarsall.bat" %VCVARS_PLATFORM%
```

Comment:
Does the VS150COMNTOOLS variable not exist?

Reply:
Ping?

Reply:
VS150COMNTOOLS  (or next version) do not exist anymore.
I tested environment content, with this :
`````PowerShell
    - ps: >-
        gci env:* | sort-object name
``
A quick query on search engines has confirm this to me.

Reply:
I took a few minutes to find info, it seems that others have complained about the lack of this envvar.  I ended up finding [this](https://developercommunity.visualstudio.com/content/problem/61913/vscommtools-no-longer-defined-by-the-installer-mak-1.html), leading me to [a blog about vswhere and VSSetup.powershell](https://devblogs.microsoft.com/setup/vswhere-available/)

I have no idea what's needed to use those on AppVeyor.  Are you willing to figure it out?  I'd suggest using them as a fallback if the VS{VSVER}0COMNTOOLS envvar doesn't exist.

Reply:
I will try to make some tests, but later in this month, as I'm going to travel.